src/utilities/strategy-helpers/deployment-helper.ts

```diff
@@ -154,7 +154,7 @@ function labelResources(files: string[], kubectl: Kubectl, label: string) {
     let workflowName = process.env.GITHUB_WORKFLOW;
     workflowName = workflowName.startsWith('.github/workflows/') ?
         workflowName.replace(".github/workflows/", "") : workflowName;
-    const labels = [`workflowFriendlyName=${workflowName}`, `workflow=${label}`];
+    const labels = [`workflowFriendlyName=${workflowName.replace(/ /g, "_")}`, `workflow=${label}`];

```

Comment:
Move the logic to a utility method `normalizeLabel`.
Can any other label have this problem?

Reply:
As of now we are labelling based on custom inputs only in this case. Other labels will not have this issue.